adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/info/AvailableReleasesResource.kt

```diff
@@ -18,14 +17,13 @@ import javax.ws.rs.core.MediaType
 @Produces(MediaType.APPLICATION_JSON)
 @Timed
 @ApplicationScoped
-@GZIP
 class AvailableReleasesResource
 @Inject
 constructor(
     private val apiDataStore: APIDataStore
 ) {
     @GET
-    @Path("/available_releases/")
+    @Path("/available_releases")
```

Comment:
RESTeasy classic ignores trailing slashes, in the definition as well as in the route matching.
Removed the trailing slashes as there are tests, that verify _only_ non-trailing slash paths.
Other option: Add both `@Path` `/` variants.

The Quarkus team will discuss whether they want to behave like RESTeasy classic: https://github.com/quarkusio/quarkus/issues/16406